### ISSUE TITLE
lindera-tantivy 0.19 doesn't compile

### ISSUE THREAD

**Issue by PSeitz** [lindera-morphology/lindera-tantivy#65]
There's a compile error due to invalid arguments
```
➜  lindera-tantivy git:(main) cargo build
    Blocking waiting for file lock on build directory
   Compiling lindera-tantivy v0.19.0 (/home/pascal/LinuxData/Development/lindera-tantivy)
error[E0061]: this function takes 4 arguments but 0 arguments were supplied
   --> src/tokenizer.rs:34:24
    |
34  |             tokenizer: LTokenizer::new()?,
    |                        ^^^^^^^^^^^^^^^-- multiple arguments are missing
    |
note: associated function defined here
   --> /home/pascal/.cargo/registry/src/github.com-1ecc6299db9ec823/lindera-0.19.5/src/tokenizer.rs:241:12
    |
241 |     pub fn new(
    |            ^^^
help: provide the arguments
    |
34  |             tokenizer: LTokenizer::new(/* lindera_core::dictionary::Dictionary */, /* Option<lindera_core::user_dictionary::UserDictionary> */, /* lindera_core::viterbi::Mode */, /* bool */)?,
    |                                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

error[E0277]: the `?` operator can only be applied to values that implement `Try`
  --> src/tokenizer.rs:34:24
   |
34 |             tokenizer: LTokenizer::new()?,
   |                        ^^^^^^^^^^^^^^^^^^ the `?` operator cannot be applied to type `lindera::tokenizer::Tokenizer`
   |
   = help: the trait `Try` is not implemented for `lindera::tokenizer::Tokenizer`

error[E0599]: no function or associated item named `with_config` found for struct `lindera::tokenizer::Tokenizer` in the current scope
  --> src/tokenizer.rs:40:36
   |
40 |             tokenizer: LTokenizer::with_config(config)?,
   |                                    ^^^^^^^^^^^ function or associated item not found in `lindera::tokenizer::Tokenizer`
```

**Comment by mosuka** [comment]
@PSeitz 
Thanks for letting me know.
I will fix it as soon as I can, so please wait a bit.

**Comment by mosuka** [comment]
@PSeitz 
I've released v0.19.1. Please try this one.
